askama_shared/src/parser.rs

```diff
@@ -128,6 +128,37 @@ pub struct CondTest<'a> {
     pub expr: Expr<'a>,
 }
 
+/// Skips over the any amout of `inner` until `end` was found.
+/// Returns a tuple of the skipped string and the found end marker.
+fn skip_till<'a, I, O>(
+    mut inner: impl FnMut(&'a [u8]) -> IResult<&'a [u8], I>,
```

Comment:
I think `take_content()` is now the only caller for `skip_till`, which means `inner` is always `anychar`. Can we simplify? Maybe even inline `skip_till()` into `take_content()`?

Reply:
The function is used in #546, too.
I removed the "inner" parameter, because it's "anychar" for both its uses.
I cleaned up the implementation a bit. I think it's much easier on the eyes now. :)